src/components/index.js

```diff
@@ -30,7 +30,7 @@ export {OverlayAction} from './overlay-action';
 export {OverlayPortal} from './overlay-portal';
 export * from './player';
 export {PrePlaybackPlayOverlay} from './pre-playback-play-overlay';
-export {Forward} from './forward';
+export {Forward, Forward as ForwardControl} from './forward';
```

Comment:
add a new line at the end of the file. where the `Rewind`

Reply:
done